core/domain/rights_manager.py

```diff
@@ -474,6 +766,18 @@ def can_unpublish(self, activity_type, activity_id):
         return self.is_moderator()
 
     def can_modify_roles(self, activity_type, activity_id):
+        """Returns whether this user has modify role right for given activity.
+
+        Args:
+            activity_type: The type of activity. Either one of these:
+                feconf.ACTIVITY_TYPE_EXPLORATION
+                feconf.ACTIVITY_TYPE_COLLECTION
+
+            activity_id: str. ID of activity.
+
+        Returns:
+            bool. Whether the user has a right to modify role of the activity.
```

Comment:
role of the --> roles for the given

(Btw, note that these are roles for editors, creators, etc. The activity itself doesn't have a role.)